### PR TITLE
Apply idea code analysis suggestions M-N

### PR DESCRIPTION
**Pull request by mocenas**
### Summary

Apply idea code suggestions. Mostly grammar fixes, updating deprecated API usage and some code simplyfication.

Please select the relevant options.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Dependency update
- [X] Refactoring
- [ ] Backport
- [ ] New scenario (non-breaking change which adds functionality)
- [ ] This change requires a documentation update
- [x] This change requires execution against OCP (use `run tests` phrase in comment)

### Checklist:
- [ ] Methods and classes used in PR scenarios are meaningful
- [ ] Commits are well encapsulated and follow [the best practices](https://cbea.ms/git-commit/)

### CODE REVIEW

**Comment by mocenas** [comment]
Issues in OCP are unrelated.

File: messaging/infinispan-grpc-kafka/README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Table of Contents
 1. [Quarkus infinispan grpc kafka](#Quarkus-infinispan-grpc-kafka)
-2. [Quarkus SSL/TLS Infinispan scenario ](#Quarkus-SSL/TLS-Infinispan-scenario )
-3. [Quarkus Grateful Shutdown for Kafka connectors](#Quarkus-Grateful-Shutdown-for-Kafka-connectors)
+2. [Kafka-client SASL](#Kafka-client-SASL)
```
Comment (gtroitsk) [inline messaging/infinispan-grpc-kafka/README.md:3]: We have SASL SSL scenario in this module, please rename

File: messaging/infinispan-grpc-kafka/README.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ Verifies SASL authentication through Quarkus Kafka client extension.
 Endpoint `SaslKafkaEndpoint` is able to produce events
 and consume events and check topics through `AdminClient` and `KafkaConsumer`.
 
-## Quarkus Infinispan scenario 
+## Quarkus Infinispan scenario
 
 ##Infinispan server SSL/TLS
```
Comment (gtroitsk) [inline messaging/infinispan-grpc-kafka/README.md:19]: As I understand Michal V recently added support to generate certs automatically, so some content in this section may be outdated

File: messaging/infinispan-grpc-kafka/README.md (hunk 1 above)
Comment (mocenas) [reply messaging/infinispan-grpc-kafka/README.md:3]: IMHO chapters in the readme does not need to to 1on1 match to the scenarios. Other sections are also not doing it, but OK I will change it.

File: messaging/infinispan-grpc-kafka/README.md (hunk 2 above)
Comment (mocenas) [reply messaging/infinispan-grpc-kafka/README.md]: I've rechecked the readme and cleaned it up a bit. There was a lot of references to stuff, that are no longer part of this module.

**Pull request merged**